dp-frontend-renderer.nomad

```diff
@@ -0,0 +1,82 @@
+job "dp-frontend-renderer" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger      = "20s"
+    max_parallel = 1
+  }
+
+  group "web" {
+    count = "{{WEB_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "web"
+    }
+
+    task "dp-frontend-renderer-web" {
+      driver = "docker"
+
+      config {
```

Comment:
Seems that this stanza is missing the `command` and `args` fields. We need to run `start-task` which will setup the environment for the task and then run the binary, e.g:

```
command = "${NOMAD_TASK_DIR}/start-task"
args = ["./dp-frontend-renderer"]
```

Same for the publishing tasks.